code/modules/reagents/newchem/drinks.dm

```diff
@@ -62,3 +62,91 @@
 	result = "jackrose"
 	required_reagents = list("applejack" = 4, "lemonjuice" = 1)
 	result_amount = 5
+
+
+// ROBOT ALCOHOL PAST THIS POINT
+// WOOO!
+
+
+/datum/reagent/ethanol/synthanol
+	name = "Synthanol"
+	id = "synthanol"
+	description = "A runny liquid with conductive capacities. Its effects on synthetics are similar to those of alcohol on organics."
+	reagent_state = LIQUID
+	color = "#1BB1FF"
+	process_flags = SYNTHETIC
+	metabolization_rate = 0.4
+	vomit_start = INFINITY		//
+	blur_start = INFINITY		//
+	pass_out = INFINITY			//INFINITY, so that IPCs don't puke and stuff
+	var/collapse_start = 200	//amount absorbed after wich mob starts collapsing
+	var/braindamage_start = 300 //amount absorbed after which mob starts taking small amount of brain damage
+
+
+/datum/chemical_reaction/synthanol
+	name = "Synthanol"
+	id = "synthanol"
+	result = "synthanol"
+	required_reagents = list("lube" = 1, "plasma" = 1, "fuel" = 1)
+	result_amount = 3
+	mix_message = "The chemicals mix to create shiny, blue substance."
+
+/datum/reagent/ethanol/synthanol/on_mob_life(var/mob/living/M as mob, var/alien)
+
+	var/d = data 
```

Comment:
It needs the d defined here, apparently. It's throwing compile errors if I remove it.